backend/application/syson-sysml-import/src/test/resources/ASTTransformerTest/convertAliasTest/model.ast.json

```diff
@@ -15,21 +15,99 @@
                   "$type": "OwningMembership",
```

Comment:
Why this file is named model.ast.json ? Shouldn't it be alias.ast.json instead?
(same remarks for all other similar cases)

Reply:
done